index.js

```diff
@@ -414,8 +417,10 @@ function assignTokens(step, worker) {
     // do not call the timer once we have reached
     // the final content item.
     if (step < 5) {
-        // update the lastSidebarLaunchTime to now
-        simpleStorage.lastSidebarLaunchTime = Date.now();
+        // update the lastSidebarCTACompleteTime to now
+        simpleStorage.lastSidebarCTACompleteTime = Date.now();
+        // note that we haven't yet shown our notification for this sidebar
+        simpleStorage.shownNotification = false;
```

Comment:
We need to also set it to false when we either (a) start the onboarding at step one (the above example) or (b) complete the current CTA and start the next timer